### DISCUSSION TITLE
Optional parameters not working for Pushover

### DISCUSSION THREAD

**Discussion by cockleburrito** [containrrr/shoutrrr#267] [Q&A] [answered]
Running Watchtower using this notification URL (masking my keys):

`pushover://shoutrrr:[API key]@[user key]/?title=Watchtower&priority=-1`

And none of the optional parameters (title and priority) are working, I'm just getting the default notifications.

Can anyone see what I'm doing wrong? Thanks!

**Comment by piksel** [comment, marked as answer]
Answered in https://github.com/containrrr/watchtower/discussions/1376
